### PR TITLE
Initial commit of README.adoc, CONTRIBUTING.adoc, and artifacts from microprofile.io

### PR DESCRIPTION
**Pull request by kwsutter**
Should we go ahead and commit this to get something readable for this repo?  Or, would it imply that we already have maven artifacts with this groupId, etc?

### CODE REVIEW

**Comment by OndrejM** [comment]
@kwsutter Eclipse IP validator has problems with commits done with the github interface - for some reason, the web interface seems to create the commits with the noreply@github.com email address which is unknown to the IP validator. It's awkward for these small commits, but I always commit and push from my local copy to set proper user, email and sign-off.

**Comment by OndrejM** [comment]
@kwsutter I think we can copy most of the pom.xml in the original repo, fixing all pre-Eclipse information and giving @ivannov, @kenfinnigan and you the attribution via the NOTICE file. Do you agree?

**Comment by kwsutter** [comment]
Yeah, I noticed that after the fact.  I'll get that cleaned up.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Eclipse MicroProfile "Bill of Materials"
```
Comment (OndrejM) [inline README.md]: We should add the copyright header as here: https://raw.githubusercontent.com/eclipse/microprofile-config/master/README.adoc

**Comment by kwsutter** [comment]
Shoot, is there an easy way to clean up that ip-validation error?  I thought a new commit from my branch would clean it up, but no such luck...  Anybody have any luck or experience with using the "git commit -s::" command to do the "sign off"?

**Comment by mikecroft** [comment]
@kwsutter you can amend and signoff the previous commit:
https://stackoverflow.com/a/15667644/212224

```
git commit --amend --signoff
```

**Comment by kwsutter** [comment]
I've tried two different times with the "git commit --amend --signoff" and I still have the ip check failure...  FYI, the reason for two attempts is that on the first one I forgot to do the "git push --force" to force the override.  But, even with that, I still have the ip check failure.  Any other ideas?  I hate to close it and start over...  Thanks!

**Comment by OndrejM** [comment]
I think the ip-validator is just broken. I don't now why it thinks that a user noreply@github.com is an author of any commits. I would close this PR and recreate it, maybe it will help.

**Comment by kwsutter** [comment]
Ondrej, I guess this ip-validation is just a warning.  We could still merge if we're sure that I am a valid committer...  :-)
https://www.eclipse.org/projects/handbook/#resources-github

**Comment by kwsutter** [comment]
Okay, let's try this again...  I'm going to close this PR down and start over to clear up the IP-Validation errors...

**Pull request closed without merging**